docs/docs/usage/build-time-compiler.md

```diff
@@ -91,6 +103,67 @@ var instance = Instance.builder(module).
         withMachineFactory(AddModule::create).
         build();
 ```
+### wasm-aot-gen Goal Configuration Parameters
```

Comment:
Is this section generated?
Can we leave an hidden note on how to regenerate?

Something like:

```
mvn help:describe -DgroupId=io.fabric8 -DartifactId=java-generator-maven-plugin -Dversion=<version> -Ddetail
```
I imagine

Reply:
Added a generated version, you did excellent work and it was looking great, but we don't want to spend time maintaining an hand crafted html version of it.
People will figure out ... and eventually improve the docs.